test/auth.test.js

```diff
@@ -77,7 +78,8 @@ describe('SolidAuthOIDC', () => {
     let auth
 
     beforeEach(() => {
-      auth = new SolidAuthOIDC()
+      localStorage.clear()
+      auth = new SolidAuthOIDC({ store: localStorage })
```

Comment:
Looks like all the `beforeEach`s do the same thing. You can combine them into a single one by wrapping all the `describe` blocks in a top-level describe.